lighthouse-core/test/config/config-test.js

```diff
@@ -365,53 +365,50 @@ describe('Config', () => {
     });
   });
 
-  describe('getAggregations', () => {
+  describe('getCategories', () => {
     it('returns the IDs & names of the aggregations', () => {
-      const runConfig = JSON.parse(JSON.stringify(defaultConfig));
-      const aggs = Config.getAggregations(runConfig);
-      assert.equal(Array.isArray(aggs), true);
-      assert.equal(aggs.length, 5, 'Found the correct number of aggregations');
-      const haveName = aggs.every(agg => agg.name.length);
-      const haveID = aggs.every(agg => agg.id.length);
-      assert.equal(haveName === haveID === true, true, 'they dont have IDs and names');
+      const categories = Config.getCategories(defaultConfig);
+      assert.equal(Array.isArray(categories), true);
+      assert.equal(categories.length, 4, 'Found the correct number of categories');
+      const haveName = categories.every(agg => agg.name.length);
+      const haveID = categories.every(agg => agg.id.length);
+      assert.equal(haveName === haveID === true, true, 'they have IDs and names');
     });
   });
 
-  describe('generateConfigOfAggregations', () => {
-    const aggregationIDs = ['perf'];
+  describe('generateConfigOfCategories', () => {
+    const categoryIds = ['performance'];
```

Comment:
nit: inheriting this, but maybe just inline this in the three tests that use it? doesn't seem helpful to have on its own